aten/src/ATen/native/mps/operations/Normalization.mm

```diff
@@ -153,12 +153,6 @@ static void get_shapes(MPSShape* input_shape_readonly,
     else
       channelsDim = num_input_dims - 1;
 
-    bool executeGatherOp = true;
-    if (self.is_contiguous(memory_format)) {
-      memory_format = MemoryFormat::Contiguous;
-      executeGatherOp = false;
-    }
-
```

Comment:
Does my assumption that the decision to gather is sufficiently handled in aten/src/ATen/native/mps/OperationUtils.mm hold?

Reply:
Looks like everything passed in CI.